moped-editor/src/views/projects/projectView/ProjectComponents/utils/map.js

```diff
@@ -84,7 +84,7 @@ export function useAppBarHeight() {
     currentToolbarMinHeight = toolbar;
   }
 
-  return currentToolbarMinHeight.minHeight;
+  return currentToolbarMinHeight?.minHeight;
```

Comment:
fixes a WSOD when resizing the window with the map to a very small size

Reply:
THANK YOU FOR FIXING THIS!!!!